neo/Network/P2P/TaskManager.cs

```diff
@@ -58,7 +59,7 @@ private void OnInternalTaskCompleted(UInt256 hash)
         {
             if (!sessions.TryGetValue(Sender, out TaskSession session))
                 return;
-            session.Tasks.Remove(hash);
+            if (hash != StateRootTaskHash) session.Tasks.Remove(hash);
```

Comment:
What's the difference?

Reply:
`StateRootTaskHash` only exits in `GlobalTasks`.

Reply:
But if it doesn't exist, `Remove()` will do nothing. So we don't need to check whether it is `StateRootTaskHash`.

Reply:
done